c_api/index_io_c.cpp

```diff
@@ -15,35 +15,40 @@
 
 using faiss::Index;
 
-int faiss_write_index(const FaissIndex *idx, FILE *f) {
-    try {
-        faiss::write_index(reinterpret_cast<const Index*>(idx), f);
-    } CATCH_AND_HANDLE
+int faiss_write_index(const FaissIndex *idx, FILE *f)
+{
+    try
+    {
+        faiss::write_index(reinterpret_cast<const Index *>(idx), f);
+    }
+    CATCH_AND_HANDLE
 }
 
-int faiss_write_index_fname(const FaissIndex *idx, const char *fname) {
-    try {
-        faiss::write_index(reinterpret_cast<const Index*>(idx), fname);
-    } CATCH_AND_HANDLE
+int faiss_write_index_fname(const FaissIndex *idx, const char *fname)
+{
+    try
+    {
+        faiss::write_index(reinterpret_cast<const Index *>(idx), fname);
+    }
+    CATCH_AND_HANDLE
 }
 
-int faiss_read_index(FILE *f, int io_flags, FaissIndex **p_out) {
-    try {
+int faiss_read_index(FILE *f, int io_flags, FaissIndex **p_out)
+{
+    try
+    {
         auto out = faiss::read_index(f, io_flags);
-        *p_out = reinterpret_cast<FaissIndex*>(out);
-    } CATCH_AND_HANDLE
+        *p_out = reinterpret_cast<FaissIndex *>(out);
+    }
+    CATCH_AND_HANDLE
 }
 
-int faiss_read_index_fname(const char *fname, int io_flags, FaissIndex **p_out) {
-    try {
+int faiss_read_index_fname(const char *fname, int io_flags, FaissIndex **p_out)
+{
+    try
+    {
         auto out = faiss::read_index(fname, io_flags);
-        *p_out = reinterpret_cast<FaissIndex*>(out);
-    } CATCH_AND_HANDLE
-}
-
-int faiss_clone_index (const FaissIndex *idx, FaissIndex **p_out) {
```

Comment:
`clone_index` still appears to be in the library, declared in [clone_index.h](https://github.com/facebookresearch/faiss/blob/master/clone_index.h).